### ISSUE TITLE
WA page update

### ISSUE THREAD

**Issue by timathomas** [miaschwing/EN_newtemplate#1]
Hey @miaschwing. This is page is great! love the stats. Couple suggestions: 

- [ ] Can you add the ERN logo with white lettering in the upper left when it's not scrolling down? (i.e., on the initial landing) Then it changes to black lettering logo when it's a white background. 
<img width="763" alt="Screenshot 2024-09-05 at 4 32 22 PM" src="https://github.com/user-attachments/assets/e3454bac-7b92-4c74-b3b6-b26f8443854b">

- [ ] Can you add the white lettering logo in the footer to replace the spelled out Eviction Research Network?

**Comment by timathomas** [comment]
- [ ] Add a link to where those stats are coming from. I think most are from the report, no?

**Comment by miaschwing** [comment]
Yes, all of the stats are coming from the report. I will make sure to mention it in the description of the section.

**Comment by miaschwing** [comment]
And I will try and get the white logo so it then changes to the black one. And I will add it in the footer.

side note: I did not know you could do comments on github! This is so cool
